gobuffalo/azuredeploy.json

```diff
@@ -37,45 +44,92 @@
       },
       "databaseAdministratorLogin": {
           "type": "String",
-          "defaultValue": "[concat('admin', parameters('name'))]",
           "metadata": {
             "description": "The user handle for the administrator of the database to be created."
           }
       },
       "databaseAdministratorLoginPassword": {
         "type": "SecureString",
-        "defaultValue": "",
+        "defaultValue": "[concat('MSFT+Buffalo-', uniqueString(resourceGroup().id), resourceGroup().location)]",

```

Comment:
this is deterministic, just use an empty string here

Reply:
Fixed locally.